sktime/forecasting/adapters/_autots.py

```diff
@@ -0,0 +1,649 @@
+        fh : guaranteed to be ForecastingHorizon or None, optional (default=None)
+            The forecasting horizon with the steps ahead  to predict.
+            Required (non-optional) here.
+        X : pd.DataFrame, optional (default=None)
+            Exogeneous time series to fit to.
+
+        Returns
+        -------
+        self : reference to self
+        """
+        # various type input indices are converted to datetime
+        # since AutoTS can only deal with dates
+        y = self._convert_input_to_date(y)
+        self._y = y
+
+        self._fh = fh
+        self._instantiate_model()
+        try:
+            self.forecaster_.fit(df=self._y)
+        except Exception as e:
+            raise e
+        return self
+
+    def _predict(
+        self,
+        fh: Union[ForecastingHorizon, None] = None,
+        X: [pd.DataFrame, None] = None,  # noqa: F841
+    ):
+        """Provide forecast at future horizon using fitted forecaster.
+
+        State required:
+            Requires state to be "fitted".
+
+        Accesses in self:
+            Fitted model attributes ending in "_"
+
+        Parameters
+        ----------
+        fh : guaranteed to be ForecastingHorizon or None, optional (default=None)
+            The forecasting horizon with the steps ahead to predict.
+        X : pd.DataFrame, optional (default=None)
+            Exogenous time series
+
+        Returns
+        -------
+        y_pred : pd.DataFrame
+            Point predictions
+        """
+        if fh is not None:
+            self._fh = fh
+
+        values = self.forecaster_.predict(
+            forecast_length=self._get_forecast_length()
+        ).forecast.values
+
+        cutoff = self._fh_cutoff_transformation(self._y)
+        values = values[self._fh.to_relative(cutoff)._values - 1]
+
+        # convert back to original index
+        row_idx: pd.Index = self._fh.to_absolute_index(self.cutoff)
+        col_idx = self._y.columns
+        y_pred = pd.DataFrame(values, index=row_idx, columns=col_idx)
+
+        return y_pred
+
+    @classmethod
+    def get_test_params(cls, parameter_set="default"):
+        """Return testing parameter settings for the estimator.
+
+        Parameters
+        ----------
+        parameter_set : str, default="default"
+            Name of the set of test parameters to return, for use in tests. If no
+            special parameters are defined for a value, will return `"default"` set.
+            There are currently no reserved values for forecasters.
+
+        Returns
+        -------
+        params : dict or list of dict, default = {}
+            Parameters to create testing instances of the class
+            Each dict are parameters to construct an "interesting" test instance, i.e.,
+            `MyClass(**params)` or `MyClass(**params[i])` creates a valid test instance.
+            `create_test_instance` uses the first (or only) dictionary in `params`
+        """
+        params = [
```

Comment:
I haven't checked myself, but if most of these values match the defaults, can you just keep the non-defaults? Otherwise if something fails, it'll be difficult to find what's causing the issues.